contents/newsletter/how-to-accidentally-kill-your-startup.md

```diff
@@ -0,0 +1,148 @@
+---
+title: How to accidentally kill your startup
```

Comment:
I don't think this headline totally fits. Since this is a bit more hot takey like collaboration sucks, we could probably go with "Prevent defense" or "Playing not to lose" as the headline, as they're more thematic and add a bit of a curiosity gap. Also, the startup didn't get killed.